src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -182,7 +188,7 @@ public override TypeKind TypeKind
         {
             get
             {
-                return TypeKind.Tuple;
+                return _underlyingType.TypeKind;
```

Comment:
Actually it may be better to dafault to a struct in broken scenarios since most common correct case will be struct.